apps/association_result_annotation/dxapp.json

```diff
@@ -124,24 +148,67 @@
     {
       "name": "out_file",
```

Comment:
"out_file" -> "out_file_tsv"
Also need to update in the app script accordingly.